src/sortStudents.ts

```diff
@@ -1,16 +1,74 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: [],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade'
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function avgGrade(student: Student): number {
+  return student.grades.reduce(
+    (prev: number, item: number) => prev + item, 0,
```

Comment:
reduce is smart enough to infer types from array which uses reduce method 
```suggestion
    (prev, item) => prev + item, 0,
```